### ISSUE TITLE
When trying to install in a go module, version 1.10 is not recognized.

### ISSUE THREAD

**Issue by jac1013** [cosmtrek/air#18]
When running
```bash
$ go get github.com/cosmtrek/air@v1.0.2 
```
It installs air successfully.

When running
```bash
$ go get github.com/cosmtrek/air@v1.10
```

I get `go get github.com/cosmtrek/air@v1.10: no matching versions for query "v1.10"
`

Am I doing something wrong here? Thanks in advance!

**Comment by cosmtrek** [comment]
This project is used as a binary command, not the third library. So you could download the binary file.

**Comment by jac1013** [comment]
Hey @cosmtrek, first of all thanks for the library, I really appreciate your work.

Now, yes, I understand that the project is used as a binary command but we still can do a `go get github.com/cosmtrek/air@v1.0.2` so go modules know that it needs to be "installed" as a dependency.

The command above works, by the way, after having the dependency in `go.mod` and building or running the tests of the project we can run `./air` in the project without setting up anything as you specify in the readme.

My point here is, why 
```bash
$ go get github.com/cosmtrek/air@v1.0.2
```
works, and...
``` bash
$ go get github.com/cosmtrek/air@v1.10
```
doesn't.

I think the reason is because tags must follow semver so they are [go modules compliant](https://github.com/golang/go/wiki/Modules#semantic-import-versioning).

Changing the version from `1.10` to `1.10.0` should solve this issue. I don't volunteer myself to do this change because it's about creating a new tag, which is something that probably only you (the owner) can do.

**Comment by cosmtrek** [comment]
Oops...It's my mistake to not follow the semver rules. I'll fix it in next release.  Thanks for your advice!

**Comment by goenning** [comment]
I've just hit the same problem, might be an option to release `1.10.1` with the same code as `1.10`.

**Comment by cosmtrek** [comment]
@goenning I've  fixed some bugs and released a new version v1.11.1 :)
